docs/faq.md

```diff
@@ -142,3 +142,23 @@ Refer to the [PSPDFKit blog post for more details.](https://pspdfkit.com/blog/20
 ```swift
 UIApplication.sharedApplication().keyWindow?.layer.speed = 100
 ```
+
+**How do I reset application state before each test using Swift?**
+
+In the application's Build Settings, set **Defines Module** to **Yes**. Create a `resetApplication()` method on the AppDelegate.
+
+In the EarlGrey test target, import the application using `@testable`. In `setUp()`, aquire a reference to the delegate then invoke `resetApplication()`.
+
+```swift
+// swift
+@testable import App
+
+class MyTests: XCTestCase {
+
+    override func setUp() {
```

Comment:
No because we want to ensure reset runs before each test run. If we had a previous run without reset, then we're already in a bad state and resetting in tearDown doesn't help us.